src/OVAL/probes/unix/linux/rpm-helper.c

```diff
@@ -24,8 +24,11 @@
 #include <config.h>
 #endif
 
-int rpmErrorCb (rpmlogRec rec, rpmlogCallbackData data)
+void rpmLibsPreload()
 {
-	dE("RPM: %s", rpmlogRecMessage(rec));
-	return RPMLOG_EXIT; // don't perform default logging
```

Comment:
This is very drastic, could we keep the Cb and set it after we are done with the preload? We still want those messages, imagine the pain of debugging any problems with the RPM probes without those messages...